cloudinit/config/cc_mounts.py

```diff
@@ -257,6 +257,7 @@ def create_swap(fname, size, method):
         except subp.ProcessExecutionError as e:
             LOG.warning(errmsg, fname, size, method, e)
             util.del_file(fname)
+            raise
```

Comment:
Why not `raise subp.ProcessExecutionError`?

Reply:
This would raise a new, unparameterised instance of `ProcessExecutionError` instead of just raising the already-existing exception, losing some stack information. Consider:

```py
def inner():
    raise Exception("some info")


def outer1():
    try:
        inner()
    except Exception:
        raise Exception


def outer2():
    try:
        inner()
    except Exception:
        raise
```

If you call `outer1()`, you see:

```pytb
Traceback (most recent call last):
  File "example.py", line 7, in outer1
    inner()
  File "example.py", line 2, in inner
    raise Exception("some info")
Exception: some info

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "example.py", line 18, in <module>
    outer1()
  File "example.py", line 9, in outer1
    raise Exception
Exception
```

vs this for `outer2()`:

```pytb
Traceback (most recent call last):
  File "example.py", line 18, in <module>
    outer2()
  File "example.py", line 14, in outer2
    inner()
  File "example.py", line 2, in inner
    raise Exception("some info")
Exception: some info
```